src/views/pages/page.njk

```diff
@@ -2,40 +2,71 @@
 {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
-{% from "pages/components/macro.njk" import multiSideBar,
-conditionalButtons %}
+{%
+  from "pages/components/macro.njk" import multiSideBar,
+  conditionalButtons
+%}
 {% set items = meta.items %}
+{% set errors = meta.errors %}
 {% set sidebar = meta.sidebar %}
 
 {% block pageTitle %}
-  {% if errorList %}
+
+  {% if errors[meta.currentPageId] %}
     Error:
   {% endif %}
   {{ pageTitle }}
 {% endblock %}
 
 {% block beforeContent %}
   {% if meta.previousPageId %}
-    {{ govukBackLink({
-      id: "backLink",
-      text: "Back",
-      href: meta.previousPageId
-    }) }}
-  {% endif %}
-  {% if errorList %}
-    {{ govukErrorSummary ({
-        "titleText": "There is a problem",
-        "errorList": errorList
-      }) }}
+    {{
+      govukBackLink({
+        id: "backLink",
+        text: "Back",
+        href: meta.previousPageId
+      })
+    }}
   {% endif %}
+
+  
 {% endblock %}
 
 {% block content %}
   <div id="content" class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds govuk-!-margin-top-2">
+      {% if errors[meta.currentPageId] %}
```

Comment:
Could we extract this into:

```
{% set currentPageError = errors[meta.currentPageId] %}
```
and then use it like below:

```
{% block pageTitle %}
  {% if currentPageError %}
    Error:
  {% endif %}
  {{ pageTitle }}
{% endblock %}

```

Reply:
Yes, much nicer. Edited.